tests/testthat/test-cross-validate.R

```diff
@@ -0,0 +1,53 @@
+context("test-cross-validate")
+
+test_that("number of crossfolds is a single integer", {
+   expect_error(
+    crossv_kfold(mtcars, k = "0"),
+    "`k` must be a single integer"
```

Comment:
When testing for error messages, the point of the text is to make sure that the correct error message is generated, not that the text is exactly correct. For that reason, you should check for the smallest possible fragment, e.g. `"single integer"` here.